src/EnergyPlus/ZoneEquipmentManager.cc

```diff
@@ -867,366 +855,202 @@ void SetUpZoneSizingArrays(EnergyPlusData &state)
-                calcZoneSizing.CoolSizingFactor = zoneSizingInput.CoolSizingFactor;
-                calcZoneSizing.AccountForDOAS = zoneSizingInput.AccountForDOAS;
-                calcZoneSizing.DOASControlStrategy = zoneSizingInput.DOASControlStrategy;
-                calcZoneSizing.DOASLowSetpoint = zoneSizingInput.DOASLowSetpoint;
-                calcZoneSizing.DOASHighSetpoint = zoneSizingInput.DOASHighSetpoint;
-                calcZoneSizing.zoneSizingMethod = zoneSizingInput.zoneSizingMethod;
-                calcZoneSizing.zoneLatentSizing = zoneSizingInput.zoneLatentSizing;
-                calcZoneSizing.zoneRHDehumidifySetPoint = zoneSizingInput.zoneRHDehumidifySetPoint;
-                calcZoneSizing.zoneRHHumidifySetPoint = zoneSizingInput.zoneRHHumidifySetPoint;
-                calcZoneSizing.zoneRHDehumidifySchIndex = zoneSizingInput.zoneRHDehumidifySchIndex;
-                calcZoneSizing.zoneRHHumidifySchIndex = zoneSizingInput.zoneRHHumidifySchIndex;
-                calcZoneSizing.ZnLatCoolDgnSAMethod = zoneSizingInput.ZnLatCoolDgnSAMethod;
-                calcZoneSizing.LatentCoolDesHumRat = zoneSizingInput.LatentCoolDesHumRat;
-                calcZoneSizing.CoolDesHumRatDiff = zoneSizingInput.CoolDesHumRatDiff;
-                calcZoneSizing.ZnLatHeatDgnSAMethod = zoneSizingInput.ZnLatHeatDgnSAMethod;
-                calcZoneSizing.LatentHeatDesHumRat = zoneSizingInput.LatentHeatDesHumRat;
-                calcZoneSizing.HeatDesHumRatDiff = zoneSizingInput.HeatDesHumRatDiff;
-            }
+
+            if (DesDayNum == 1 && ZoneSizNum == 0) { // LKL I think this is sufficient for warning -- no need for array
+                if (!state.dataGlobal->isPulseZoneSizing) {
+                    ShowWarningError(
+                        state,
+                        format("SetUpZoneSizingArrays: Sizing for Zone=\"{}\" will use Sizing:Zone specifications listed for Zone=\"{}\".",
+                               state.dataZoneEquip->ZoneEquipConfig(CtrlZoneNum).ZoneName,
+                               zoneSizingInput.ZoneName));
+                }
+            }
+
+            zoneSizing.ZnCoolDgnSAMethod = zoneSizingInput.ZnCoolDgnSAMethod;
+            zoneSizing.ZnHeatDgnSAMethod = zoneSizingInput.ZnHeatDgnSAMethod;
+            zoneSizing.CoolDesTemp = zoneSizingInput.CoolDesTemp;
+            zoneSizing.HeatDesTemp = zoneSizingInput.HeatDesTemp;
+            zoneSizing.CoolDesTempDiff = zoneSizingInput.CoolDesTempDiff;
+            zoneSizing.HeatDesTempDiff = zoneSizingInput.HeatDesTempDiff;
+            zoneSizing.CoolDesHumRat = zoneSizingInput.CoolDesHumRat;
+            zoneSizing.HeatDesHumRat = zoneSizingInput.HeatDesHumRat;
+            zoneSizing.CoolAirDesMethod = zoneSizingInput.CoolAirDesMethod;
+            zoneSizing.HeatAirDesMethod = zoneSizingInput.HeatAirDesMethod;
+            zoneSizing.InpDesCoolAirFlow = zoneSizingInput.DesCoolAirFlow;
```

Comment:
Is this and the same for Heat the only place in which these structs are different?